core/sdk-core/src/test/java/software/amazon/awssdk/core/internal/batchmanager/BatchManagerTest.java

```diff
@@ -159,14 +157,31 @@ public void scheduleFiveMessagesWithEachThreadToDifferentDestinations() {
         checkThreadedResponses(requests, responses, sendRequestFutures);
         executorService.shutdownNow();
     }
+
+    @Test
+    public void periodicallySendMessagesWithTenThreadsToSameDestination() {
```

Comment:
This might have been more of a debugging test to see how periodically sending requests would work (since it would involve a combination of scheduling and manually flushing the buffers).

Reply:
Hmm, sounds like this is not needed here if it's only used for debugging purpose

Reply:
Should I still include this test since no other test verifies how periodically sending requests might work with the BatchManager?